docs/examples/redundancey_fault_tolerance.rst

```diff
@@ -0,0 +1,80 @@
+.. _examples_redundancy:
+
+Redundancy and Fault Tolerance
+==============================
+
+*Fast DDS* allows more than one publisher to write data on the same Topic.
+All publishers may have the same relevance, or one publisher can be set as the primary publisher and keep the rest as
+secondary publishers.
+In that case, only the main publisher can send data to the subscribers.
+
+The Ownership QoS determines if the Topic (Shape) is owned by a single publisher or not.
+There are two ownership options: ``SHARE`` or EXLUSIVE ownership.
+The value of a publisher's strength can be set using the ``EXCLUSIVE`` configuration.
+Therefore, only the publisher with the highest strength can send data on this Topic.
+If ``SHARE`` is selected, all the publishers can write data at the same time.
+
+In this test, two publishers with ``EXCLUSIVE`` ownership in SQUARE Shape, and one subscriber with
+``EXCLUSIVE`` ownership at the same Shape, will be created.
+
+Step-by-step example implementation
+-----------------------------------
+
+First, launch two instances and create a publisher in each of them:
+
+1. Create a red square publisher:
+
+   - Start eProsima Shapes Demo (this instance will be referred to as *Instance1*).
+   - Click on Publish.
+   - Select SQUARE option for Shape and RED for Color.
+   - Change the History field from 6 to 1.
+   - Select ``EXCLUSIVE``.
+   - Set Strength to 1.
+   - Set Size to 15.
+
+2. Create a red square publisher:
+
+   - Start eProsima Shapes Demo (this instance will be referred to as *Instance2*).
+   - Click on Publish.
+   - Select SQUARE option for Shape and RED for Color.
+   - Change the History field from 6 to 1.
+   - Select ``EXCLUSIVE``.
+   - Set Strength 2.
+   - Set Size to 30.
+
+A small red square on Instance1 and a big red square on Instance2 should be displayed.
+
+.. image:: /01-figures/test5_2.png
```

Comment:
This image shows instance3 even though the instructions have not created it yet. I'd add a note to explain what the 3rd window is